src/ontology/genepio-edit.owl

```diff
@@ -2439,7 +2440,7 @@ SubClassOf(obo:APOLLO_SV_00000021 obo:STATO_0000047)
 
 SubClassOf(obo:APOLLO_SV_00000022 obo:STATO_0000047)
 
-# Class: obo:APOLLO_SV_00000031 (simulated population)
+# Class: obo:APOLLO_SV_00000031 (simulation of population)
```

Comment:
Not clear why this change occurred. I didn't touch this term and it isn't present in the new canada_covid19.owl import file. When I view it in Protégé it shows two labels `simulated population` and `simulation of population`, both with the `general_import.owl` listed as the source.

Any thoughts @ddooley ?

Reply:
A fetch of general_ontofox.txt that created general_ontofox.owl probably updated the label of the term.  This change is just a comment (on subclass axiom below) to label so of no concern, easy to accept.

SubClassOf(obo:APOLLO_SV_00000031 obo:IAO_0000027)